model_analyzer/config/input/config_command_profile.py

```diff
@@ -221,6 +222,14 @@ def _fill_config(self):
                 default_value=DEFAULT_GPUS,
                 description="List of GPU UUIDs to be used for the profiling. "
                 "Use 'all' to profile all the GPUs visible by CUDA."))
+        self._add_config(
+            ConfigField(
+                'skip_summary_reports',
```

Comment:
We used to have this option but it was removed in this commit: https://github.com/triton-inference-server/model_analyzer/commit/93991747f579fe26b0adc914d6de8228e63df0d3#diff-bfa8f90a391c1ffdc5f96eac924441f3941ab2aeb9d7784b89ab3f503cd37c8dL156-L158 Maybe we can use the same name now that we are re-adding it?

Reply:
We don't want the same name as the default behavior is opposite. We want to create summary reports/tables by default, as that is the common case usage.

We could call this '--skip-summarize', but I'd much rather be slightly more verbose and more explicit with the naming of the flag. I like '--skip-summary-reports' and unless there are strong objections I'd like to keep this new name.

Reply:
Sounds good. Resolving.